rdrf/rdrf/testing/behaviour/features/steps.py

```diff
@@ -944,3 +908,24 @@ def sidebar_contains_link_in_section(step, sec, name):
             "Could not find a link to %s in section %s in sidebar"
             % (name, sec)
         )
+
+
+@step('should see (survey name|communication type) option "(.*)"')
+def proms_checks(step, which, name):
+    assert world.browser.current_url.endswith("/clinicalproms"),\
+        "Not on PROMS request page!"
+    xp = (
+        "//label[@for='id_%s']/following-sibling::select"
```

Comment:
might be good to have a helper function find_option(dropdown_label, option_text)